iocore/cache/test/main.cc

```diff
@@ -24,9 +24,23 @@
 #define CATCH_CONFIG_MAIN
 #include "main.h"
 
+#include <unistd.h>
+#include <filesystem>
+
 #define THREADS 1
 #define DIAGS_LOG_FILE "diags.log"
 
+// Create a new temp directory and return it
+std::string
+temp_prefix()
+{
+  char buffer[PATH_MAX];
+  snprintf(buffer, sizeof(buffer), "%s/cachetest.XXXXXX", getenv("TMPDIR") ?: "/tmp");
```

Comment:
Interesting - that's a gcc extension I've never seen used before. It seems to work in clang as well, although not one of 8 different C++ reference documents I checked mention it. Looks like it works on all the compilers we support (although MSVC doesn't support it).

Reply:
Wow, I thought being able to omit the consequent was standard now but I also can't find it.  It's certainly handy.